eap-session-replication/ocp-template/eap71-basic-s2i.json

```diff
@@ -0,0 +1,432 @@
+{
```

Comment:
Openshift template file which can be removed.

Reply:
I have removed the Openshift file